docs/tfimport/README.md

```diff
@@ -57,14 +57,22 @@ differences between the configs and the actual infrastructure.
 $ tfimport -h
 Usage:
   -dry_run
-     Run in dry-run mode, which only prints the import commands without running them.
+        Run in dry-run mode, which only prints the import commands
+        without running them.
   -input_dir string
-     Path to the directory containing Terraform configs. (default ".")
+        Path to the directory containing Terraform configs. (default ".")
   -interactive
-     Interactively ask for user input when import information cannot be
-     automatically determined. (default true)
+        Interactively ask for user input when import information cannot be
+        automatically determined. (default true)
+  -resource_types value
```

Comment:
change the type to string or list of string?

Reply:
it seems --dry_run and --interactive did not have type specified, we can also add those